docs/HowToUse.md

```diff
@@ -0,0 +1,28 @@
+# How to use?
+
+Let's compile an example by using ONE-vscode.
+
+## Install the toolchain
+
+First, you need the ONE toolchain. Let's install the ONE toolchain. (You should install the in-house backend to follow this.)
+
+<img src="https://user-images.githubusercontent.com/7223627/174947559-71ab213c-68c4-4700-bcec-e55f6a2203bf.gif" width=600 />
+
+## Prepare a NN model and a config file
+
+Second, you need a NN model and a config file to run compiling. Now you can get one from [res/modelDir/truediv](../res/modelDir/truediv/). Add it to your workspace. Now you can see a model and a config.
+
+<img src="https://user-images.githubusercontent.com/10216715/174798969-eee44fea-bd71-4e6a-8e2c-9e1de37ad74a.gif" width=600 />
+
+## Compile models
+
+Last, you can compile the model by pushing Run button.
+
+<img src="https://user-images.githubusercontent.com/10216715/174796457-4dae4a77-04e1-4e5c-9453-77ebfb65182a.gif" width=600 />
```

Comment:
Now we have 'run'(play) button near on the configuration file. We can change the gif later if may.

Reply:
How about uploading this comment? I'll change it.

Reply:
I failed to upload the gif file from this window desktop... I think I can do it in the office, l will try it later.